lib/common/graph.js

```diff
@@ -204,6 +204,24 @@ class ConcertoGraphVisitor extends DiagramVisitor {
         parameters.graph.addVertex(scalarDeclaration.getFullyQualifiedName());
     }
 
+    /**
+     * Visitor design pattern
+     * @param {Decorator} decorator - the object being visited
+     * @param {Object} parameters  - the parameter
+     * @protected
+     */
+    visitDecorator(decorator, parameters) {
+        const parent = decorator.getParent();
+        const modelFile = parent.getModelFile();
+        decorator.getArguments()?.forEach(argument => {
+            const fqn = modelFile.getFullyQualifiedTypeName(argument.name);
```

Comment:
Does this work even if the argument isn't a Type?

Reply:
I.e. what happens if the argument is a random string?

Reply:
Yes, this is accounted for. The `getFullyQualifiedTypeName` returns null in such a scenario as it would not find the import.